### PR TITLE
Pipepanic

### PR DESCRIPTION
**Pull request by lorglas**
bump revision, added deskbar symlink

### CODE REVIEW

**Comment by Begasus** [comment]
I see a revision bump here, not a version bump?

**Comment by lorglas** [comment]
I needed to a a new patches. How can i change the patch set.

**Comment by lorglas** [comment]
I have add the new patchset, please check it.

File: games-puzzle/pipepanic/pipepanic-0.1.3.recipe [hunk 1]
```diff
@@ -4,7 +4,7 @@ different shaped pipes together as possible within the time given."
 HOMEPAGE="http://www.users.waitrose.com/~thunor/pipepanic/"
 COPYRIGHT="2006 TheGreenKnight"
 LICENSE="GNU GPL v2"
-REVISION="7"
+REVISION="9"
```
Comment (Begasus) [inline games-puzzle/pipepanic/pipepanic-0.1.3.recipe:7]: I think this  should be revision=8?

Comment (lorglas) [reply games-puzzle/pipepanic/pipepanic-0.1.3.recipe]: Change to 8 done.

**Pull request merged**